src/meta/src/hummock/manager/mod.rs

```diff
@@ -607,15 +595,15 @@ where
             .id_gen_manager()
             .generate::<{ IdCategory::HummockCompactionTask }>()
             .await?;
+        let group_config = self
+            .compaction_group_manager()
+            .compaction_group(compaction_group_id)
+            .await
+            .ok_or(Error::InvalidCompactionGroup(compaction_group_id))?;
         if !compaction
             .compaction_statuses
             .contains_key(&compaction_group_id)
```

Comment:
Can we add comment/doc about lifecycle of both compaction group entry in CompactStatus and HummockVersion, like when they are (lazily)initialized/destroyed, when it's an expected behavior or a fatal bug if trying to visit a missing group entry?

Reply:
> Can we add comment/doc about lifecycle of both compaction group entry in CompactStatus and HummockVersion, like when they are (lazily)initialized/destroyed, when it's an expected behavior or a fatal bug if trying to visit a missing group entry?

https://singularity-data.quip.com/iohfAKFRfxtu/Consistent-among-CompactionGroupManager-HummockVersion-and-CompactStatus-in-meta